src/main/java/com/google/impactdashboard/server/api_utilities/IamBindingRetriever.java

```diff
@@ -0,0 +1,121 @@
+package com.google.impactdashboard.server.api_utilities;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.iam.v1.Iam;
+import com.google.api.services.iam.v1.IamScopes;
+import com.google.api.services.iam.v1.model.Role;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.audit.AuditLog;
+import com.google.impactdashboard.data.IAMBindingDatabaseEntry;
+import com.google.logging.v2.LogEntry;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.Timestamp;
+import com.google.protobuf.Value;
+
+import java.util.AbstractMap.SimpleImmutableEntry;
+import java.io.ByteArrayInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.checkerframework.checker.nullness.Opt.get;
+
+/**
+ * Class that counts number of members in each IAM role and determines the total number of bindings
+ */
+public class IamBindingRetriever {
+
+  private final Map<String, Integer> mapRoleToNumberOfMembers;
+  private final Iam iamService;
+
+  protected IamBindingRetriever( Map<String, Integer> mapRoleToNumberOfMembers, Iam iamService) {
+    this.mapRoleToNumberOfMembers = mapRoleToNumberOfMembers;
+    this.iamService = iamService;
+  }
+
+  /**
+   * Static factory for creating a new instance of IamBindingRetriever.
+   * @return new Instance of IamBindingRetriever
+   */
+  public static IamBindingRetriever create() throws Exception {
+    GoogleCredentials credentials;
+    try {
+      credentials = GoogleCredentials
+          // The path will be changed to use the constants class when it is merged into main
+          .fromStream(new FileInputStream("/usr/local/google/home/ionis/Documents/credentials.json"))
```

Comment:
Can you move this path to a configuration file somewhere in the app?

It looks like it currently won't work on anyone's computer but yours.